src/darwin/CHIPTool/CHIPTool/View Controllers/Bindings/BindingsViewController.m

```diff
@@ -132,7 +133,9 @@ - (void)_clearTextFields
 
 - (IBAction)bind:(id)sender
 {
-    int nodeId = [_nodeIDTextField.text intValue];
+    uint64_t nodeId;
```

Comment:
Can we make that a `chip::NodeId`?

Reply:
I think we are not exposing CHIP data structures in the iOS framework APIs.